package.json

```diff
@@ -31,7 +31,7 @@
     "react-router-dom": "^4.2.2",
     "react-scripts": "1.1.4",
     "react-select": "^2.0.0-beta.7",
-    "react-tooltip": "^3.6.0",
+    "react-tooltip": "^3.8.4",
```

Comment:
Why did you change the version?

Reply:
I didn't know react-tooltip was already installed. I'll revert to the previous version if it is okay.